app/javascript/packages/document-capture/components/in-person-call-to-action.tsx

```diff
@@ -23,7 +23,7 @@ function InPersonCallToAction({ altHeading, altPrompt, altButtonText }: InPerson
       <h2 id={`in-person-cta-heading-${instanceId}`} className="margin-y-2">
         {altHeading || t('in_person_proofing.headings.cta')}
       </h2>
-      <p>{altPrompt || t('in_person_proofing.body.cta.prompt_detail')}</p>
+      <p>{altPrompt}</p>
```

Comment:
Currently `altPrompt` is set as optional for this component. If a value isn't provided, we won't show any content here. This departs from the existing pattern of fallback values for `altHeading`, `altPrompt` and `altButtonText`. 

Rather than making `altPrompt` required and removing the fallback to the (now updated) `in_person_proofing.body.cta.prompt_detail`, could we instead leave this file as-is and remove unnecessary props in the components calling this one?